typescript-test-samples/launchdarkly-lambda-dynamodb/cdk/resources/lambdas/getFlagsWithDynamo/index.ts

```diff
@@ -0,0 +1,54 @@
+import { Handler, APIGatewayEvent } from "aws-lambda";
+// The LaunchDarkly Node SDKs
+// These needs to be npm installed or added via the Lambda Layer
+import LaunchDarkly from "launchdarkly-node-server-sdk";
+// The SDK add-on for DynamoDB support
+import { DynamoDBFeatureStore } from "launchdarkly-node-server-sdk-dynamodb";
+
+export const handler: Handler = async (event: APIGatewayEvent) => {
+  // this establishes the settings for caching in Dynamo
+  // for details, check the documentation at https://docs.launchdarkly.com/sdk/features/storing-data/dynamodb
+  const store = DynamoDBFeatureStore(process.env.DYNAMODB_TABLE as string, {
```

Comment:
Please add environment variable in the CDK template (unless I'm missing it there?)